settings/resource_default_namespace_settings_test.go

```diff
@@ -0,0 +1,215 @@
+package settings
+
+import (
+	"testing"
+
+	"github.com/databricks/databricks-sdk-go/apierr"
+	"github.com/databricks/databricks-sdk-go/service/settings"
+	"github.com/databricks/terraform-provider-databricks/qa"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestQueryCreateDefaultNameSetting(t *testing.T) {
```

Comment:
Can we add an internal/acceptance test that uses the TF template? We can run them against a test shard until we enable the UC settings public API in prod. @hectorcast-db